src/introduction.md

```diff
@@ -3,13 +3,12 @@
 This book is the primary reference for the Rust programming language.
 It provides three kinds of material:
 
-  - Chapters that informally describe each language construct and their use.
-  - Chapters that informally describe the memory model, concurrency model, runtime services, linkage model, and debugging facilities.
+  - Chapters that describe each language construct and their use.
+  - Chapters that describe the memory model, concurrency model, runtime services, linkage model, and debugging facilities.
   - Appendix chapters providing rationale and references to languages that influenced the design.
 
-> [!WARNING]
-> This book is incomplete. Documenting everything takes a while.
-> See the [GitHub issues] for what is not documented in this book.
```

Comment:
It may make sense to explicitly acknowledge that we say next to nothing about the operational semantics of the language ("which exact set of behaviors can my program have") -- at least the kind of people I hang out with would expect that to "obviously" be part of a document that "describes the language".

In which sense do we even describe the memory model or concurrency model currently?